src/compositor/lipstickcompositor.cpp

```diff
@@ -359,6 +375,24 @@ void LipstickCompositor::onSurfaceDying()
     }
 }
 
+void LipstickCompositor::initialize()
+{
+    m_displayState = new MeeGo::QmDisplayState(this);
+    MeeGo::QmDisplayState::DisplayState displayState = m_displayState->get();
+    reactOnDisplayStateChanges(displayState);
+    connect(m_displayState, SIGNAL(displayStateChanged(MeeGo::QmDisplayState::DisplayState)), this, SLOT(reactOnDisplayStateChanges(MeeGo::QmDisplayState::DisplayState)));
```

Comment:
Minor formatting issue: this line and lines 389 and 392 are rather long, line breaks would improve readability.